Project.toml

```
@@ -21,6 +21,8 @@ julia = "^1"
 
 [extras]
 Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
+CondaPkg = "992eb4ea-22a4-4c89-a5bb-47a3300528ab"
+PythonCall = "6099a3de-0909-46bc-b1f4-468b9a2dfc0d"
 
 [targets]
-test = ["Test"]
+test = ["Test", "CondaPkg", "PythonCall"]
```

Similarly, let's keep order alphabetical.
```suggestion
test = ["CondaPkg", "PythonCall", "Test"]
```

---

And can you add compat bounds for these test deps?